communicator/config.go

```diff
@@ -165,6 +165,12 @@ type SSH struct {
 	SSHBastionCertificateFile string `mapstructure:"ssh_bastion_certificate_file"`
 	// `scp` or `sftp` - How to transfer files, Secure copy (default) or SSH
 	// File Transfer Protocol.
+	//
+	// NOTE: for guests using Windows with Win32-OpenSSH v9.1.0.0p1-Beta, scp
+	// (the default protocol for copying data) returns an error since the MOTW
+	// cannot be set, and the command ends-up with a non-zero error code, making
+	// transfers fail in this case. This can be solved by overriding the protocol
+	// with SFTP instead.
```

Comment:
```suggestion
	// NOTE: Guests using Windows with Win32-OpenSSH v9.1.0.0p1-Beta, scp
	// (the default protocol for copying data) returns a a non-zero error code since the MOTW
	// cannot be set, which cause any file transfer to fail. As a workaround you can override the transfer protocol
	// with SFTP instead `ssh_file_transfer_protocol = "sftp"`.
```

Reply:
Sounds good to me, I folded this one in, and regenerated the docs, I'll merge this PR if/when the tests pass